user_install.sh

```diff
@@ -60,10 +60,10 @@ MILRC_FILE=$MIL_CONFIG_DIR/milrc
 #======================#
 
 # Install no project by default, the user must select one
-SELECTED=false
-INSTALL_SUB=false
-INSTALL_PRO=false
-INSTALL_NAV=false
+SELECTED="false"
```

Comment:
I don't remember us having a separate user_install.sh. Can you explain this division?